flashcards/web/package.json

```diff
@@ -15,19 +15,25 @@
         "watch": "vitest watch"
     },
     "dependencies": {
+        "@emotion/react": "^11.10.6",
+        "@emotion/styled": "^11.10.6",
+        "@fontsource/roboto": "^4.5.8",
+        "@mui/material": "^5.11.15",
+        "@mui/styled-engine-sc": "^5.11.11",
         "@tanstack/react-query": "^4.26.1",
```

Comment:
i think u should also add @mui/icons-material

Reply:
done :)